nemo/collections/asr/models/ssl_models.py

```diff
@@ -50,7 +50,23 @@ def list_available_models(cls) -> Optional[PretrainedModelInfo]:
         Returns:
             List of available pre-trained models.
         """
-        return []
+        results = []
+
+        model = PretrainedModelInfo(
+            pretrained_model_name="ssl_en_conformer_large",
+            description="For details about this model, please visit https://ngc.nvidia.com/catalog/models/nvidia:nemo:ssl_en_conformer_large",
+            location="https://api.ngc.nvidia.com/v2/models/nvidia/nemo/ssl_en_conformer_large/versions/1.10.1/files/ssl_en_conformer_large.nemo",
```

Comment:
What's version 0.10.1?

Reply:
do you mean 1.10.1 ? 
1.10.0 - model trained only with contrastive loss
1.10.1 - model trained contrastive + mlm loss. 
we are using versioning to indicate better models of same (dataset, model ) configuration

Reply:
So how do you access the conformer large 1.10.0 model ? You have links only for 1.10.1

Reply:
following the convention of other models, `from_pretrained` gets the "best" conformer large ssl model (which is 1.10.1 in this case). other versions can be downloaded via versions tab in model card.